### DISCUSSION TITLE
Is HVM going to be a target?

### DISCUSSION THREAD

**Discussion by Diegovsky** [uwu-tech/Kind#322] [General]
I think they go hand in hand with eachother but I haven't seen any mention of it anywhere. It would be pretty cool to have a fast and expressive language such as Kind running on HVM

**Comment by steinerkelvin** [comment]
Yes, that's the plan!

**Reply by Diegovsky** [reply]
Nice! :D

**Comment by o1lo01ol1o** [comment]
@quleuber Hi, do you know the current status of Kind1 -> HVM?  Has this been started or has the priority shifted to Kind2 -> HVM?  Where would one look to contribute to either?
